### DISCUSSION TITLE
Sample Images

### DISCUSSION THREAD

**Discussion by nawnie** [TheLastBen/fast-stable-diffusion#445] [Ideas]
More of a request then an idea, any chance you would be willing to add the option to generate Sample images while training. adding the lines back into the accelerate does not work (figured itd be worth a try) obviously this would slow things down so an option to be able to not generate would be nice as well, We can always load a ckpt from the last save and open another Vmachine... but thats one less Vmachine we can train on :D

**Comment by TheLastBen** [comment]
You can load an intermediary checkpoint simply by typing its path in the testing cell

**Reply by nawnie** [reply]
but that requires 2 machines or consstant stopping, however if we know hey at 3500 its produced good images i can stop it and use that checkpoint instead of letting it go however many steps.

again this will slow things down but having the option could help reduce over trainging and having to load up multiple .ckpt to test, instead you can skip intill it starts looking similar in images.

either way thank you again having a ton of fun with this.
